### PR TITLE
GH Contrib

### PR DESCRIPTION
**Pull request by nvarner**
Change the contributing guidelines and remove the PR template

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,68 +1,32 @@
-# Contributing to the FTC SDK
+# Contributing to this app
 
-The following is a set of guidelines for contributing the FIRST FTC SDK.  The FTC Technology Team welcomes suggestions for improvements to core software, ideas for new features, requests for built-in support of new sensors, and well written bug reports.
+## First thing you need: learn Git
```
Comment (itsmerachel) [inline .github/CONTRIBUTING.md:3]: _hahahahah_

**Review by itsmerachel — APPROVED** [review]
why,,, did you change this anyway just curious?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (nvarner) [reply .github/CONTRIBUTING.md:3]: I didn't like the idea that the contributing guidelines were there every time I opened a PR, and we didn't make them, so I changed them.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,68 +1,32 @@
...
 
-#### How Do I Submit A (Good) Bug Report?
+## Making a change
 
-Bugs are tracked as GitHub issues. Create an issue on ftctechnh/ftc_app and provide the following information.
-Explain the problem and include additional details to help maintainers reproduce the problem:
+### Branching
 
-- Use a clear and descriptive title for the issue to identify the problem.
+Wneh you want to add or remove something from the app, the first step is to create a new branch. Make sure to name it something descriptive, for example `travis-no-emulator` instead of `changes`. You are reccomended to use GitHub's built in functionality to make the branch. Use `git pull` to update your local repository, then check out the branch you've just created.
```
Comment (itsmerachel) [inline .github/CONTRIBUTING.md:14]: im calling you out but at the same time im also calling me out

**Pull request merged**